tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -3187,6 +3187,122 @@ public function testContentTypeSwitch()
         $this->mapper->save($testSiteData, 'internal-link', 'sulu_io', 'en', 1, true, $uuid);
     }
 
+    /**
+     * It should delete a node which has children with history.
+     * It should not throw an exception
```

Comment:
The test should not differ from the specification in the future, if it does then the behavior has changed.

`testDeleteWithChildHistrory` says:

- We are testing delete
- We are deleting something with a child history
OR
- We are deleting something with a child and something about history

That is not at all clear imo. and indeed it took me sometime to work out what this test did, as it has taken me lots and lots of time to understand what many tests do in Sulu (not a criticism of Sulu, but just generally).